### PR TITLE
 [DEVOPS-2374] -- add go, prefixout

### PR DESCRIPTION
**Pull request by hinling-whistle**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -85,6 +85,13 @@ RUN mkdir -p /usr/local/bin && \
     cd /tmp && \
     rm -rf /tmp/build
 
+# Install prefixout from Trevor
+RUN set -exv \
+  && apk add go \
+  && go get -u -x -v github.com/akatrevorjay/prefixout
```
Comment (arwalls) [inline Dockerfile]: Wouldn't we fork that to a whistle repo?

Comment (hinling-whistle) [reply Dockerfile]: sure

Comment (hinling-whistle) [reply Dockerfile]: updated

**Review by arwalls — APPROVED** [review]
LGTM

**Review by akatrevorjay — CHANGES_REQUESTED** [review]
These changes should likely go in the former build image.

Doing it this way adds the entire go runtime as well as all source code and it's dependencies to the final image.

(All you need to do is copy it in to the `./bin` folder for it to go into your PATH.)

Alternatively you can install the binary release, but I don't care if we compile it or not, there are pros and cons to both.

**Comment by hinling-whistle** [comment]
what do you mean by "former build image"? I tested it and it's in the final build image but I can move it down to the end if that's what you mean?

I prefer to compile it with the image in case we need to move this to some other crazy base container. 

But you are correct that we don't need the whole go runtime. I will remove it after copying the binary.

**Comment by akatrevorjay** [comment]
^ answered in person

**Review by akatrevorjay — APPROVED** [review]
lgtm

**Pull request merged**